src/style.css

```diff
@@ -1 +1,2 @@
-/* add styles here */
+@import 'stars.css';
+@import 'reset.css';
```

Comment:
It's generally a good practice to import reset styles before other styles to ensure that the reset is applied first. Consider switching the order of these imports.